Fluid/Utils/HexUtilities.cs

```diff
@@ -0,0 +1,24 @@
+﻿#if NET6_0_OR_GREATER
+
+namespace Fluid.Utils;
+
+public class HexUtilities
```

Comment:
maybe `internal static`?

Reply:
I want to use it in the benchmark, and I can't get it to work because of public key (trust me I know how it works, but maybe I don't). I may just remove the benchmarks now that I have the result.